meinberlin/templates/account/login.html

```diff
@@ -1,44 +1,51 @@
 {% extends "account/base.html" %}
 {% load i18n account socialaccount %}
 
-{% block head_title %}{% translate "Login" %}{% endblock %}
+{% block head_title %}
+    {% translate "Login" %}
+{% endblock head_title %}
 
 {% block content %}
-    <h1>{% translate "Login" %}</h1>
-
-    <p>{% blocktranslate %}If you have not created an account yet, then please <a href="{{ signup_url }}">register</a> first.{% endblocktranslate %}</p>
-
-    <form method="POST" action="{% url 'account_login' %}" class="u-spacer-bottom-double">
-        {% if form.non_field_errors %}
-            <ul class="errorlist" aria-live="assertive" aria-atomic="true">
-            {% for error in form.non_field_errors %}
-                <li>{{ error|escape }}</li>
-            {% endfor %}
-            </ul>
-        {% endif %}
-        {% csrf_token %}
-
-        <div class="form-group form-group--narrow">
-            {% include 'meinberlin_contrib/includes/form_field.html' with field=form.login %}
-            {% include 'meinberlin_contrib/includes/form_field.html' with field=form.password %}
-            {% include 'meinberlin_contrib/includes/form_checkbox_field.html' with field=form.remember %}
-        </div>
-
-        {% if redirect_field_value %}
-            <input type="hidden" name="{{ redirect_field_name }}" value="{{ redirect_field_value }}"/>
+    <div class="account__wrapper">
+        <h1>{% translate "Login" %}</h1>
+
+        <form method="post" action="{% url 'account_login' %}" class="form--base">
+            {% if form.non_field_errors %}
+                <ul class="errorlist" aria-live="assertive" aria-atomic="true">
+                    {% for error in form.non_field_errors %}
```

Comment:
I couldn't find anything in the designs regarding the non-field error messages but should they also use the `message message--error` class? Currently they are shown as a list item: 
 * Der Anmeldename und/oder das Passwort sind leider falsch.